dropwizard-app/src/main/java/org/coner/boundary/CompetitionGroupBoundary.java

```diff
@@ -0,0 +1,34 @@
+package org.coner.boundary;
+
+import org.coner.core.domain.CompetitionGroup;
+
+/**
+ *
+ */
+public class CompetitionGroupBoundary extends AbstractBoundary<
+        org.coner.api.entity.CompetitionGroup,
+        CompetitionGroup,
+        org.coner.hibernate.entity.CompetitionGroup> {
+
+    @Override
+    protected EntityMerger<org.coner.api.entity.CompetitionGroup, CompetitionGroup> buildApiToDomainMerger() {
```

Comment:
I added support for this to the ReflectionEntityMerger with test coverage for valid String<->Enum transformations and invalid transformations. See ReflectionEntityMergerTest